tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -993,9 +1011,11 @@ function test_do_not_sync_non_public_post_types_filtered_post_content() {
 		$this->sender->do_sync();
 		$synced_post = $this->server_replica_storage->get_post( $post_id );
 
-		$this->assertEquals( '', $synced_post->post_content_filtered );
-		$this->assertEquals( '', $synced_post->post_excerpt_filtered );
+		// Clean up.
+		unregister_post_type( 'non_public' );
 
+		$this->assertSame( '', $synced_post->post_content_filtered );
```

Comment:
@obenland do you mind outlining why the change to **assertSame**?

assertSame -> Asserts that two objects refer to the same object

Reply:
PHPCS wouldn't let me commit with `assertEquals` 🤷

Reply:
interesting. checking the branch out to see what it is yelling about :) Thanks for taking the initiative to clean-up these tests we greatly appreciate it.

Reply:
Interesting, so assertSame is preferred over assertEquals 🎩 another good addition of today I learned.

> assertEquals accepts many non-string values, please use strict alternatives like assertSame